### PR TITLE
Update statedb Get to return error

### PR DESCRIPTION
**Pull request by arnaucube**

### CODE REVIEW

File: vochain/process.go [hunk 1]
```diff
@@ -64,9 +64,15 @@ func (v *State) Process(pid []byte, isQuery bool) (*models.Process, error) {
...
 	v.RLock()
 	if isQuery {
-		processBytes = v.Store.ImmutableTree(ProcessTree).Get(pid)
+		processBytes, err = v.Store.ImmutableTree(ProcessTree).Get(pid)
+		if err != nil {
+			return nil, err
+		}
 	} else {
-		processBytes = v.Store.Tree(ProcessTree).Get(pid)
+		processBytes, err = v.Store.Tree(ProcessTree).Get(pid)
```
Comment (p4u) [inline vochain/process.go]: Use `if processBytes, err = .... ; err != nil {} `

Comment (p4u) [reply vochain/process.go]: Same for all other cases

Comment (arnaucube) [reply vochain/process.go]: Right! Updated.

Comment (arnaucube) [reply vochain/process.go]: Would love to have a linter for this hehe

**Review by p4u — APPROVED** [review]

**Pull request merged**